### DISCUSSION TITLE
Inbox tweaks

### DISCUSSION THREAD

**Discussion by deandratb** [AprilSylph/XKit-Rewritten#248] [Suggestion Box]
I just tried the extension out and I love it. I'd be very grateful to have the inbox options back (or anything similiar) of 'tag an ask with sender's username,' 'assign custom tag for anonymous asks,' 'assign custom tag to all published asks,' and an 'outbox' section to keep track of sent mail--if any of those are possible. Thanks!

**Comment by AprilSylph** [comment]
Outbox will be handled by a different project (https://github.com/AprilSylph/Outbox-for-Tumblr) which I would like to ship to Firefox soon, then to Chrome later.

Automatic tagging of any kind will require a more robust system for detecting the post editor's current context. It was easier in XInbox for XKit 7 on Tumblr's legacy site since XInbox straight-up intercepted the answer form, which isn't possible now.
